src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -37,12 +37,13 @@ public TagHelperDescriptorProvider(IEnumerable<TagHelperDescriptor> descriptors)
         /// </summary>
         /// <param name="tagName">The name of the HTML tag to match. Providing a '*' tag name
         /// retrieves catch-all <see cref="TagHelperDescriptor"/>s (descriptors that target every tag).</param>
+        /// <param name="attributeNames">Attributes the HTML element must contain to match.</param>
```

Comment:
nit: wording like this works in `[TargetElement]` as well (and is a bit simpler than what you have there now)